www/v1/stock2shop/scripts/stock2shop/README.md

```diff
@@ -0,0 +1,23 @@
+# SCRIPTS
```

Comment:
probably remove this, maybe include in app as a script?
Useful for app... more confusing in this repo.

Reply:
Removed voJson from connector.
Added to app under the scripts directory.